challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  ['dog'] if input.include? 'dog'
```

Comment:
Interesting approach, definitely works for finding new `Enum` methods :) - I think though some more idiomatic methods or expressions might be the best option going forward 👍